tests/foreman/sys/test_rename.py

```diff
@@ -57,7 +60,7 @@ def test_positive_rename_satellite(self, module_org, module_product):
                 resync repos, republish CVs and re-register hosts
             7. Create new entities (run end-to-end test from robottelo)
 
-        :BZ: 1469466
+        :BZ: 1469466, 1897360
```

Comment:
there are two `:BZ:` docstring token for same test, on line no. 47 and 63.

Reply:
my apologies, good catch! Strange that pre-commit checks didn't catch this